dev-libs/cowsql/cowsql-1.15.7-r2.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Embeddable, replicated and fault tolerant SQL engine (fork of dqlite)"
+HOMEPAGE="https://cowsql.dev/ https://github.com/cowsql/cowsql"
+SRC_URI="https://github.com/cowsql/cowsql/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3-with-linking-exception"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-db/sqlite:3
+	dev-libs/libuv:=
+	>=dev-libs/raft-0.18.1:="
+DEPEND="${RDEPEND}
+	test? ( dev-libs/raft[lz4,test] )"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=( "${FILESDIR}"/dqlite-1.12.0-disable-werror.patch )
+
+src_prepare() {
+	default
+	sed -i -e 's/float_t/float_cow/g' src/lib/serialize.h || die
```

Comment:
Please use the sed across the codebase and send the patch upstream first.

Reply:
Patch send upstream (https://github.com/cowsql/cowsql/pull/37). Using `cowsql_float` instead of `float_cow` and using a patch instead of using sed.